### ISSUE TITLE
Sync crashes when computer locked 

### ISSUE THREAD

**Issue by StavenCross** [phw198/OutlookGoogleCalendarSync#692]
:memo: Use the below template to report your issue.  
:bulb: Upgrade to the latest alpha release to check if the problem is already fixed.  
:paperclip: To aid investigation, provide your OGcalsync.log file ([see instructions](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-problems#locating-your-ogcalsynclog-file)).  

----

**OGCS Version**: v2.7.8.0  
**Installed or Portable**: Installed

**Problem Description**  
When my computer is locked (Windows 10), and I come back to it, I'll see little menu bars on the bottom left of the screen, it looks like its opening a msgbox, but if you click on the menu bars to maximize them, they just disapear. Once these come up, OGCS crashes. 

Occasionally, I've also noticed that before the sync, it will ask me to select the shared microsoft calendar that I'm syncing from - my guess is that when the computer locks, OGCS is trying to contact my outlook installation and is not able to because its locked and fails.

**Steps to Reproduce the Issue**  
1. Open OGCS
2. Schedule a sync
3. Lock computer

**Any other information**  
log file:https://gist.github.com/StavenCross/ac2ff6b6ca33f7f12282b1016366a026

**Comment by phw198** [comment]
The logfile you provided doesn't cover any crashes or even starting up of OGCS. Did you expect it to?

I'd need the logfile just after one of these crashes to see what's happening. Screenshots might also be helpful too...

Thanks!

**Comment by StavenCross** [comment]
Paul,

Apologies, I didn't realize that the log file wasn't persistent.

I've identified the issue further as the following :

Enable auto sync every hour
After N minutes of inactivity, the computer goes to sleep
Computer goes into a hybernation mode
Wake up computer
Log back in
Ogcs has times out and is now unresponsive.

Looks like if it attempts a sync while the pc is hybernating it becomes
unresponsive.

**Comment by phw198** [comment]
Yes, log files are persistent though it looks like you only uploaded the one dated 2019-3-28, and that shows syncs every hour from midnight to 10am - so no hibernating happening there.

But are you saying that you have to kill the OGCS process because it's totally unresponsive? That obviously wouldn't get logged, but I'd expect to see you starting up OGCS again...

I'm not quite sure how OGCS can try to sync while the computer is hibernated - the computer doesn't process _anything_ when hibernated; the RAM is written to disk. I also always hibernate my work computer and never see this problem.

**Comment by StavenCross** [comment]
That's correct. I have to kill it. Just above the task bar there are what
look like minimized message boxes (though I know that isn't really possible
either)

I hear what you're saying though. Perhaps this doesn't have to do with the
sync while hybernating, but just the hybernating in general? Ill try
unsetting the sync and see if it still becomes unresponsive
